module_4/task_7/Array.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace task_7
+{
+    class Array
+    {
+        public enum Direction 
+        {
+            Decrease = 1,
+            Increase
+        }
+
+        private int[] SortDecrease(int[] array)
```

Comment:
Better create a one method, which the contains bool parameter which represented sort order, for example bool isDesc.